.github/workflows/add_link_checker.yml

```diff
@@ -0,0 +1,20 @@
+# This is the config of the link checker GitHub Action (i.e., https://github.com/marketplace/actions/link-checker).
+
+on: [push]
+name: Check markdown links
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Link Checker
+        uses: peter-evans/link-checker@v1
+      - name: Create Issue From File
+        uses: peter-evans/create-issue-from-file@v2
+        with:
+          title: Link Checker Report
+          content-filepath: ./link-checker/out.md
+          labels: report, automated issue
+          assignees: praneesha
+      - name: Fail if there were link errors
+        run: exit ${{ steps.lc.outputs.exit_code }}
```

Comment:
Is this intended to have this along with removing the `Extending with compiler extensions` page?